src/lib/dev-env/dev-env-core.js

```diff
@@ -43,7 +41,6 @@ const containerImages = {
 };
 
 export const defaults = {
```

Comment:
Maybe this constants (same for the above) could be extracted into a constants file?

Reply:
Yep good idea.